core/field_dropdown.js

```diff
@@ -300,6 +296,10 @@ Blockly.FieldDropdown.prototype.setText = function(text) {
   this.updateTextNode_();
 
   if (this.textElement_) {
+    // Update class for dropdown text.
```

Comment:
Is this ever cleared?  If not, aren't you just adding ' blocklyDropdownText' once for every time the text is changed?